internal/indexer/indexer.go

```diff
@@ -476,17 +476,38 @@ func ServiceAccountsByOIDCClaims(obj client.Object) []string {
 			if rawClaimValue == "" {
 				continue
 			}
-			claimValues := strings.Split(rawClaimValue, ",")
-			for _, e := range claimValues {
+			for e := range strings.SplitSeq(rawClaimValue, ",") {
 				if claimValue := strings.TrimSpace(e); claimValue != "" {
 					refinedClaimValues = append(refinedClaimValues, FormatClaim(rawClaimName, claimValue))
 				}
 			}
 		}
+		if strings.HasPrefix(annotationKey, rbacapi.AnnotationKeyOIDCClaims) {
+			claimsMap := make(map[string]any)
```

Comment:
DM'ed you about this. You _can_ change this to `map[string][]string` as you suggested earlier.